docs/skillmap/collector-themed/collectort1.md

```diff
@@ -293,13 +293,13 @@ forever(function () {
 
 ---
 
-Now you have the start of your very own collector game. Give it a try in the simulator!
+Now you have the start of your very own collector game. Give it a try on the game screen!
 
 When you're done playing, click **Finish** to return to the main page where you can share your game with family and friends!
 
 
 ```package
-arcade-background-scroll=github:riknoll/arcade-background-scroll
+arcade-background-scroll=github:microsoft/arcade-background-scroll/
```

Comment:
does it work still with the trailing slashes on these? probably should I suppose

Reply:
Good question!  That will teach me to copy and paste.  In the meantime, let's let it go through and see if it works, then I'll fix it if it causes any issues.

Reply:
Yeah, looks like it loads fine in tutorial tool so shouldn't be an issue~